### ISSUE TITLE
Add Japanese translations for Japanese subdivisions

### ISSUE THREAD

**Issue by hiroki225** [hexorx/countries#343]
add Japanese translation for Japanese prefectures

```yaml
'01':
  unofficial_names:
  - Hokkaidô
  - Hokkaido
  translations:
    en: Hokkaido
  geo:
    latitude: 44
    longitude: 143
    min_latitude: 
    min_longitude: 
    max_latitude: 
    max_longitude: 
  name: Hokkaido
```
to

```yaml
'01':
  unofficial_names:
  - Hokkaidô
  - Hokkaido
  translations:
    en: Hokkaido
    ja: 北海道
  geo:
    latitude: 44
    longitude: 143
    min_latitude: 
    min_longitude: 
    max_latitude: 
    max_longitude: 
  name: Hokkaido
```

**Comment by rposborne** [comment]
Could you please submit a PR?

**Comment by rposborne** [comment]
Closed by #344
